2018-09-14 07:03:28.678 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 07:03:28.682 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 07:04:28.678 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 07:04:28.682 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 07:05:22.701 2459 DEBUG oslo_concurrency.lockutils [req-9bd7ca2b-b4ef-453c-818e-0adcc1a98e62 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:05:23.159 2459 DEBUG neutron.agent.dhcp.agent [req-2101e8bc-6fcc-43c2-b233-bb9616ba83d5 - - - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:05:23.161 2459 DEBUG neutron.agent.linux.utils [req-2101e8bc-6fcc-43c2-b233-bb9616ba83d5 - - - - -] Unable to access /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:05:23.161 2459 DEBUG neutron.agent.linux.dhcp [req-2101e8bc-6fcc-43c2-b233-bb9616ba83d5 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-cd5b296e-c635-4f46-bf81-ce5baa0224c5 on network cd5b296e-c635-4f46-bf81-ce5baa0224c5 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1272
2018-09-14 07:05:23.162 2459 DEBUG neutron.agent.linux.dhcp [req-2101e8bc-6fcc-43c2-b233-bb9616ba83d5 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-cd5b296e-c635-4f46-bf81-ce5baa0224c5 on network cd5b296e-c635-4f46-bf81-ce5baa0224c5 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1293
2018-09-14 07:05:25.944 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:25.965 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:25.978 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:25.995 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.007 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.022 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', 'link', 'set', 'tap66fe03bf-ab', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.047 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-o', 'link', 'show', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.064 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-14 07:05:26.088 2459 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap66fe03bf-ab) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-14 07:05:26.099 2459 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-09-14 07:05:26.101 2459 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap66fe03bf-ab) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-14 07:05:26.102 2459 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Port, col_values=(('tag', 4095),), record=tap66fe03bf-ab) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-14 07:05:26.103 2459 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=2): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'66fe03bf-ab1f-4df7-84e2-161a00954044', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:d9:a0:c3'})), record=tap66fe03bf-ab) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-14 07:05:26.131 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap66fe03bf-ab', 'address', 'fa:16:3e:d9:a0:c3'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.143 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.156 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap66fe03bf-ab', 'netns', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.180 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', 'link', 'set', 'tap66fe03bf-ab', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.192 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', 'link', 'set', 'tap66fe03bf-ab', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.204 2459 DEBUG oslo_concurrency.lockutils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Acquired semaphore "iptables-qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:05:26.206 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.227 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.244 2459 DEBUG neutron.agent.linux.iptables_manager [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-09-14 07:05:26.246 2459 DEBUG oslo_concurrency.lockutils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Releasing semaphore "iptables-qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:05:26.248 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-o', 'link', 'show', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.260 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', 'addr', 'show', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.272 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'addr', 'add', '11.100.1.2/24', 'scope', 'global', 'dev', 'tap66fe03bf-ab', 'brd', '11.100.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.285 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.297 2459 DEBUG neutron.agent.linux.dhcp [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Setting IPv4 gateway for dhcp netns on net cd5b296e-c635-4f46-bf81-ce5baa0224c5 to 11.100.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1169
2018-09-14 07:05:26.299 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.100.1.1', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.309 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-6', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.324 2459 DEBUG neutron.agent.linux.dhcp [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Cleaning stale devices for network cd5b296e-c635-4f46-bf81-ce5baa0224c5 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1374
2018-09-14 07:05:26.325 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.336 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.353 2459 DEBUG neutron.agent.linux.dhcp [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:626
2018-09-14 07:05:26.355 2459 DEBUG neutron.agent.linux.dhcp [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/leases with contents:
1537009526 fa:16:3e:d9:a0:c3 11.100.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:654
2018-09-14 07:05:26.356 2459 DEBUG neutron.agent.linux.dhcp [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Building host file: /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 07:05:26.357 2459 DEBUG neutron.agent.linux.dhcp [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Done building host file /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 07:05:26.358 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Unable to access /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:05:26.359 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host', '--addn-hosts=/var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap66fe03bf-ab', '--dhcp-range=set:tag0,11.100.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.384 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:05:26.385 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:05:26.386 2459 DEBUG neutron.agent.linux.external_process [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:05:26.387 2459 DEBUG oslo_concurrency.lockutils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:05:26.395 2459 DEBUG oslo_concurrency.lockutils [req-9bd7ca2b-b4ef-453c-818e-0adcc1a98e62 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:05:26.875 2459 DEBUG neutron.agent.dhcp.agent [req-53b18e9d-98b0-492f-9eb5-7c216881a6bd - - - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:05:26.876 2459 DEBUG neutron.agent.linux.dhcp [req-53b18e9d-98b0-492f-9eb5-7c216881a6bd - - - - -] Building host file: /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 07:05:26.883 2459 DEBUG neutron.agent.linux.dhcp [req-53b18e9d-98b0-492f-9eb5-7c216881a6bd - - - - -] Done building host file /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 07:05:26.889 2459 DEBUG neutron.agent.linux.utils [req-53b18e9d-98b0-492f-9eb5-7c216881a6bd - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '10860'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.903 2459 DEBUG neutron.agent.linux.dhcp [req-53b18e9d-98b0-492f-9eb5-7c216881a6bd - - - - -] Reloading allocations for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 07:05:26.904 2459 DEBUG neutron.agent.linux.utils [req-53b18e9d-98b0-492f-9eb5-7c216881a6bd - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.921 2459 DEBUG neutron.agent.linux.utils [req-53b18e9d-98b0-492f-9eb5-7c216881a6bd - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-6', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.935 2459 DEBUG neutron.agent.linux.utils [req-53b18e9d-98b0-492f-9eb5-7c216881a6bd - - - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:05:26.936 2459 DEBUG neutron.agent.linux.utils [req-53b18e9d-98b0-492f-9eb5-7c216881a6bd - - - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:05:26.937 2459 DEBUG neutron.agent.linux.external_process [req-53b18e9d-98b0-492f-9eb5-7c216881a6bd - - - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:05:26.938 2459 DEBUG oslo_concurrency.lockutils [req-53b18e9d-98b0-492f-9eb5-7c216881a6bd - - - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:05:26.939 2459 DEBUG oslo_concurrency.lockutils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:05:26.940 2459 DEBUG neutron.agent.dhcp.agent [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-14T11:05:23Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-cd5b296e-c635-4f46-bf81-ce5baa0224c5, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'de9372e2-cead-4dac-bbc2-d42ce8fe0702', u'ip_address': u'11.100.1.2'}], id=66fe03bf-ab1f-4df7-84e2-161a00954044, mac_address=fa:16:3e:d9:a0:c3, name=, network_id=cd5b296e-c635-4f46-bf81-ce5baa0224c5, port_security_enabled=False, project_id=ec596172ce524fd08b255a4f057a7009, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=ec596172ce524fd08b255a4f057a7009, updated_at=2018-09-14T11:05:24Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:410
2018-09-14 07:05:26.941 2459 DEBUG oslo_concurrency.lockutils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:05:26.942 2459 DEBUG oslo_concurrency.lockutils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:05:26.943 2459 INFO neutron.agent.dhcp.agent [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-14T11:05:23Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-cd5b296e-c635-4f46-bf81-ce5baa0224c5, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'de9372e2-cead-4dac-bbc2-d42ce8fe0702', u'ip_address': u'11.100.1.2'}], id=66fe03bf-ab1f-4df7-84e2-161a00954044, mac_address=fa:16:3e:d9:a0:c3, name=, network_id=cd5b296e-c635-4f46-bf81-ce5baa0224c5, port_security_enabled=False, project_id=ec596172ce524fd08b255a4f057a7009, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=ec596172ce524fd08b255a4f057a7009, updated_at=2018-09-14T11:05:25Z
2018-09-14 07:05:26.944 2459 DEBUG neutron.agent.dhcp.agent [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:05:26.946 2459 DEBUG neutron.agent.linux.dhcp [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Building host file: /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 07:05:26.947 2459 DEBUG neutron.agent.linux.dhcp [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Done building host file /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 07:05:26.951 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '10860'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.965 2459 DEBUG neutron.agent.linux.dhcp [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Reloading allocations for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 07:05:26.967 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:26.982 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-6', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:27.003 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:05:27.005 2459 DEBUG neutron.agent.linux.utils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:05:27.005 2459 DEBUG neutron.agent.linux.external_process [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:05:27.006 2459 DEBUG oslo_concurrency.lockutils [req-33eb3733-3640-4566-a517-9e9be0b85c94 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:05:28.678 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 07:05:28.681 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 07:05:59.781 2459 DEBUG oslo_concurrency.lockutils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:05:59.785 2459 INFO neutron.agent.dhcp.agent [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-14T11:05:58Z, description=, device_id=5393f307-8fbf-4044-9fc4-9d4d4d84926a, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'de9372e2-cead-4dac-bbc2-d42ce8fe0702', u'ip_address': u'11.100.1.1'}], id=482e364a-32ae-48d8-9f2a-948e1eb36e25, mac_address=fa:16:3e:d8:24:a4, name=, network_id=cd5b296e-c635-4f46-bf81-ce5baa0224c5, port_security_enabled=False, project_id=ec596172ce524fd08b255a4f057a7009, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=ec596172ce524fd08b255a4f057a7009, updated_at=2018-09-14T11:05:59Z
2018-09-14 07:05:59.787 2459 DEBUG neutron.agent.dhcp.agent [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:05:59.788 2459 DEBUG neutron.agent.linux.dhcp [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Building host file: /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 07:05:59.792 2459 DEBUG neutron.agent.linux.dhcp [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Done building host file /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 07:05:59.797 2459 DEBUG neutron.agent.linux.utils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '10860'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:59.813 2459 DEBUG neutron.agent.linux.dhcp [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Reloading allocations for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 07:05:59.815 2459 DEBUG neutron.agent.linux.utils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:59.832 2459 DEBUG neutron.agent.linux.utils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-6', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:05:59.849 2459 DEBUG neutron.agent.linux.utils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:05:59.850 2459 DEBUG neutron.agent.linux.utils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:05:59.851 2459 DEBUG neutron.agent.linux.external_process [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:05:59.852 2459 DEBUG oslo_concurrency.lockutils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:06:01.076 2459 DEBUG oslo_concurrency.lockutils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:06:01.077 2459 INFO neutron.agent.dhcp.agent [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-14T11:05:58Z, description=, device_id=5393f307-8fbf-4044-9fc4-9d4d4d84926a, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'de9372e2-cead-4dac-bbc2-d42ce8fe0702', u'ip_address': u'11.100.1.1'}], id=482e364a-32ae-48d8-9f2a-948e1eb36e25, mac_address=fa:16:3e:d8:24:a4, name=, network_id=cd5b296e-c635-4f46-bf81-ce5baa0224c5, port_security_enabled=False, project_id=ec596172ce524fd08b255a4f057a7009, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=ec596172ce524fd08b255a4f057a7009, updated_at=2018-09-14T11:05:59Z
2018-09-14 07:06:01.080 2459 DEBUG neutron.agent.dhcp.agent [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:06:01.082 2459 DEBUG neutron.agent.linux.dhcp [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Building host file: /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 07:06:01.085 2459 DEBUG neutron.agent.linux.dhcp [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Done building host file /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 07:06:01.089 2459 DEBUG neutron.agent.linux.utils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '10860'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:01.105 2459 DEBUG neutron.agent.linux.dhcp [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Reloading allocations for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 07:06:01.107 2459 DEBUG neutron.agent.linux.utils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:01.125 2459 DEBUG neutron.agent.linux.utils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-6', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:01.140 2459 DEBUG neutron.agent.linux.utils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:06:01.142 2459 DEBUG neutron.agent.linux.utils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:06:01.142 2459 DEBUG neutron.agent.linux.external_process [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:06:01.143 2459 DEBUG oslo_concurrency.lockutils [req-e9733a69-5140-4c2b-af39-60170673837d 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:06:04.138 2459 DEBUG oslo_concurrency.lockutils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:06:04.139 2459 INFO neutron.agent.dhcp.agent [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-14T11:05:58Z, description=, device_id=5393f307-8fbf-4044-9fc4-9d4d4d84926a, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'de9372e2-cead-4dac-bbc2-d42ce8fe0702', u'ip_address': u'11.100.1.1'}], id=482e364a-32ae-48d8-9f2a-948e1eb36e25, mac_address=fa:16:3e:d8:24:a4, name=, network_id=cd5b296e-c635-4f46-bf81-ce5baa0224c5, port_security_enabled=False, project_id=ec596172ce524fd08b255a4f057a7009, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=ec596172ce524fd08b255a4f057a7009, updated_at=2018-09-14T11:06:00Z
2018-09-14 07:06:04.141 2459 DEBUG neutron.agent.dhcp.agent [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:06:04.143 2459 DEBUG neutron.agent.linux.dhcp [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Building host file: /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 07:06:04.149 2459 DEBUG neutron.agent.linux.dhcp [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Done building host file /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 07:06:04.155 2459 DEBUG neutron.agent.linux.utils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '10860'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:04.172 2459 DEBUG neutron.agent.linux.dhcp [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Reloading allocations for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 07:06:04.175 2459 DEBUG neutron.agent.linux.utils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:04.200 2459 DEBUG neutron.agent.linux.utils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-6', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:04.213 2459 DEBUG neutron.agent.linux.utils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:06:04.214 2459 DEBUG neutron.agent.linux.utils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:06:04.215 2459 DEBUG neutron.agent.linux.external_process [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:06:04.216 2459 DEBUG oslo_concurrency.lockutils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:06:05.147 2459 DEBUG oslo_concurrency.lockutils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:06:05.148 2459 INFO neutron.agent.dhcp.agent [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-14T11:05:58Z, description=, device_id=5393f307-8fbf-4044-9fc4-9d4d4d84926a, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'de9372e2-cead-4dac-bbc2-d42ce8fe0702', u'ip_address': u'11.100.1.1'}], id=482e364a-32ae-48d8-9f2a-948e1eb36e25, mac_address=fa:16:3e:d8:24:a4, name=, network_id=cd5b296e-c635-4f46-bf81-ce5baa0224c5, port_security_enabled=False, project_id=ec596172ce524fd08b255a4f057a7009, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=ec596172ce524fd08b255a4f057a7009, updated_at=2018-09-14T11:06:04Z
2018-09-14 07:06:05.149 2459 DEBUG neutron.agent.dhcp.agent [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:06:05.150 2459 DEBUG neutron.agent.linux.dhcp [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Building host file: /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 07:06:05.152 2459 DEBUG neutron.agent.linux.dhcp [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Done building host file /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 07:06:05.154 2459 DEBUG neutron.agent.linux.utils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '10860'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:05.169 2459 DEBUG neutron.agent.linux.dhcp [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Reloading allocations for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 07:06:05.171 2459 DEBUG neutron.agent.linux.utils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:05.185 2459 DEBUG neutron.agent.linux.utils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-6', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:05.214 2459 DEBUG neutron.agent.linux.utils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:06:05.215 2459 DEBUG neutron.agent.linux.utils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:06:05.216 2459 DEBUG neutron.agent.linux.external_process [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:06:05.216 2459 DEBUG oslo_concurrency.lockutils [req-90262b16-60c4-413f-ad42-e5723ec46224 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:06:11.928 2459 DEBUG oslo_concurrency.lockutils [req-3d98c28a-b768-43d5-951b-ea273448055e 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Acquired semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:06:11.928 2459 DEBUG oslo_concurrency.lockutils [req-3d98c28a-b768-43d5-951b-ea273448055e 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Releasing semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:06:15.138 2459 DEBUG oslo_concurrency.lockutils [req-4a7f8cdf-d24e-4018-88b7-9b3362d66058 - - - - -] Acquired semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:06:15.140 2459 DEBUG oslo_concurrency.lockutils [req-4a7f8cdf-d24e-4018-88b7-9b3362d66058 - - - - -] Releasing semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:06:15.802 2459 DEBUG oslo_concurrency.lockutils [req-4a7f8cdf-d24e-4018-88b7-9b3362d66058 - - - - -] Acquired semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:06:15.808 2459 DEBUG oslo_concurrency.lockutils [req-4a7f8cdf-d24e-4018-88b7-9b3362d66058 - - - - -] Releasing semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:06:28.679 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 07:06:28.684 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 07:06:31.826 2459 DEBUG oslo_concurrency.lockutils [req-c423a76a-06da-4cab-99a2-0fecc7c840a9 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:06:31.827 2459 INFO neutron.agent.dhcp.agent [req-c423a76a-06da-4cab-99a2-0fecc7c840a9 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-14T11:06:30Z, description=, device_id=c230dbbd-561d-425d-bf77-66c79be2a5db, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'de9372e2-cead-4dac-bbc2-d42ce8fe0702', u'ip_address': u'11.100.1.7'}], id=0e75b47f-7879-4d21-99e7-3219f08a08e0, mac_address=fa:16:3e:bb:10:c4, name=, network_id=cd5b296e-c635-4f46-bf81-ce5baa0224c5, port_security_enabled=True, project_id=ec596172ce524fd08b255a4f057a7009, revision_number=3, security_groups=[u'abf59d60-6558-44d1-ba4c-ad3835777042'], status=DOWN, tags=[], tenant_id=ec596172ce524fd08b255a4f057a7009, updated_at=2018-09-14T11:06:31Z
2018-09-14 07:06:31.828 2459 DEBUG neutron.agent.dhcp.agent [req-c423a76a-06da-4cab-99a2-0fecc7c840a9 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:06:31.830 2459 DEBUG neutron.agent.linux.dhcp [req-c423a76a-06da-4cab-99a2-0fecc7c840a9 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Building host file: /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 07:06:31.834 2459 DEBUG neutron.agent.linux.dhcp [req-c423a76a-06da-4cab-99a2-0fecc7c840a9 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Done building host file /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 07:06:31.841 2459 DEBUG neutron.agent.linux.utils [req-c423a76a-06da-4cab-99a2-0fecc7c840a9 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '10860'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:31.860 2459 DEBUG neutron.agent.linux.dhcp [req-c423a76a-06da-4cab-99a2-0fecc7c840a9 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Reloading allocations for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 07:06:31.861 2459 DEBUG neutron.agent.linux.utils [req-c423a76a-06da-4cab-99a2-0fecc7c840a9 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:31.875 2459 DEBUG neutron.agent.linux.utils [req-c423a76a-06da-4cab-99a2-0fecc7c840a9 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-6', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:31.889 2459 DEBUG neutron.agent.linux.utils [req-c423a76a-06da-4cab-99a2-0fecc7c840a9 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:06:31.890 2459 DEBUG neutron.agent.linux.utils [req-c423a76a-06da-4cab-99a2-0fecc7c840a9 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:06:31.890 2459 DEBUG neutron.agent.linux.external_process [req-c423a76a-06da-4cab-99a2-0fecc7c840a9 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:06:31.891 2459 DEBUG oslo_concurrency.lockutils [req-c423a76a-06da-4cab-99a2-0fecc7c840a9 4c9d738ac49a44de954282a01551494e ec596172ce524fd08b255a4f057a7009 - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:06:34.002 2459 DEBUG oslo_concurrency.lockutils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:06:34.003 2459 INFO neutron.agent.dhcp.agent [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-14T11:06:30Z, description=, device_id=c230dbbd-561d-425d-bf77-66c79be2a5db, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'de9372e2-cead-4dac-bbc2-d42ce8fe0702', u'ip_address': u'11.100.1.7'}], id=0e75b47f-7879-4d21-99e7-3219f08a08e0, mac_address=fa:16:3e:bb:10:c4, name=, network_id=cd5b296e-c635-4f46-bf81-ce5baa0224c5, port_security_enabled=True, project_id=ec596172ce524fd08b255a4f057a7009, revision_number=4, security_groups=[u'abf59d60-6558-44d1-ba4c-ad3835777042'], status=DOWN, tags=[], tenant_id=ec596172ce524fd08b255a4f057a7009, updated_at=2018-09-14T11:06:32Z
2018-09-14 07:06:34.004 2459 DEBUG neutron.agent.dhcp.agent [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:06:34.005 2459 DEBUG neutron.agent.linux.dhcp [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Building host file: /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 07:06:34.007 2459 DEBUG neutron.agent.linux.dhcp [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Done building host file /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 07:06:34.010 2459 DEBUG neutron.agent.linux.utils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '10860'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:34.023 2459 DEBUG neutron.agent.linux.dhcp [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Reloading allocations for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 07:06:34.024 2459 DEBUG neutron.agent.linux.utils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:34.038 2459 DEBUG neutron.agent.linux.utils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-6', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:34.052 2459 DEBUG neutron.agent.linux.utils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:06:34.052 2459 DEBUG neutron.agent.linux.utils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:06:34.053 2459 DEBUG neutron.agent.linux.external_process [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:06:34.053 2459 DEBUG oslo_concurrency.lockutils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:06:34.535 2459 DEBUG oslo_concurrency.lockutils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:06:34.536 2459 INFO neutron.agent.dhcp.agent [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-14T11:06:30Z, description=, device_id=c230dbbd-561d-425d-bf77-66c79be2a5db, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'de9372e2-cead-4dac-bbc2-d42ce8fe0702', u'ip_address': u'11.100.1.7'}], id=0e75b47f-7879-4d21-99e7-3219f08a08e0, mac_address=fa:16:3e:bb:10:c4, name=, network_id=cd5b296e-c635-4f46-bf81-ce5baa0224c5, port_security_enabled=True, project_id=ec596172ce524fd08b255a4f057a7009, revision_number=5, security_groups=[u'abf59d60-6558-44d1-ba4c-ad3835777042'], status=DOWN, tags=[], tenant_id=ec596172ce524fd08b255a4f057a7009, updated_at=2018-09-14T11:06:34Z
2018-09-14 07:06:34.538 2459 DEBUG neutron.agent.dhcp.agent [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:06:34.539 2459 DEBUG neutron.agent.linux.dhcp [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Building host file: /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 07:06:34.542 2459 DEBUG neutron.agent.linux.dhcp [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Done building host file /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 07:06:34.546 2459 DEBUG neutron.agent.linux.utils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '10860'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:34.561 2459 DEBUG neutron.agent.linux.dhcp [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Reloading allocations for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 07:06:34.562 2459 DEBUG neutron.agent.linux.utils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:34.574 2459 DEBUG neutron.agent.linux.utils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-6', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:06:34.588 2459 DEBUG neutron.agent.linux.utils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:06:34.589 2459 DEBUG neutron.agent.linux.utils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:06:34.590 2459 DEBUG neutron.agent.linux.external_process [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:06:34.591 2459 DEBUG oslo_concurrency.lockutils [req-2686c25a-4b17-4c1b-acfd-5de62722bc41 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:07:00.267 2459 DEBUG oslo_concurrency.lockutils [req-e62dad41-70af-46c1-96a7-c0c9a908f632 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:07:00.268 2459 DEBUG oslo_concurrency.lockutils [req-e62dad41-70af-46c1-96a7-c0c9a908f632 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Releasing semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:07:01.356 2459 DEBUG oslo_concurrency.lockutils [req-e62dad41-70af-46c1-96a7-c0c9a908f632 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:07:01.357 2459 DEBUG oslo_concurrency.lockutils [req-e62dad41-70af-46c1-96a7-c0c9a908f632 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Releasing semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:07:28.679 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 07:07:28.682 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 07:08:28.679 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 07:08:28.682 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 07:09:28.680 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 07:09:28.684 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 07:10:28.683 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 07:10:28.688 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.006s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 07:10:46.299 2459 DEBUG oslo_concurrency.lockutils [req-1b678b35-e008-4e52-affa-0b7d28dcf893 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:10:46.300 2459 DEBUG neutron.agent.dhcp.agent [req-1b678b35-e008-4e52-affa-0b7d28dcf893 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:10:46.302 2459 DEBUG neutron.agent.linux.utils [req-1b678b35-e008-4e52-affa-0b7d28dcf893 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'dhcp_release', 'tap66fe03bf-ab', '11.100.1.7', 'fa:16:3e:bb:10:c4'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:10:46.326 2459 DEBUG neutron.agent.linux.dhcp [req-1b678b35-e008-4e52-affa-0b7d28dcf893 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Building host file: /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 07:10:46.340 2459 DEBUG neutron.agent.linux.dhcp [req-1b678b35-e008-4e52-affa-0b7d28dcf893 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Done building host file /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 07:10:46.358 2459 DEBUG neutron.agent.linux.utils [req-1b678b35-e008-4e52-affa-0b7d28dcf893 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '10860'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:10:46.370 2459 DEBUG neutron.agent.linux.dhcp [req-1b678b35-e008-4e52-affa-0b7d28dcf893 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Reloading allocations for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 07:10:46.371 2459 DEBUG neutron.agent.linux.utils [req-1b678b35-e008-4e52-affa-0b7d28dcf893 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:10:46.386 2459 DEBUG neutron.agent.linux.utils [req-1b678b35-e008-4e52-affa-0b7d28dcf893 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-6', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:10:46.399 2459 DEBUG neutron.agent.linux.utils [req-1b678b35-e008-4e52-affa-0b7d28dcf893 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:10:46.404 2459 DEBUG neutron.agent.linux.utils [req-1b678b35-e008-4e52-affa-0b7d28dcf893 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:10:46.405 2459 DEBUG neutron.agent.linux.external_process [req-1b678b35-e008-4e52-affa-0b7d28dcf893 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:10:46.407 2459 DEBUG oslo_concurrency.lockutils [req-1b678b35-e008-4e52-affa-0b7d28dcf893 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:11:28.683 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 07:11:28.688 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 07:11:36.146 2459 DEBUG oslo_concurrency.lockutils [req-0fedb962-284b-4bb5-9a08-43deb7994c12 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:11:36.148 2459 DEBUG neutron.agent.dhcp.agent [req-0fedb962-284b-4bb5-9a08-43deb7994c12 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:11:36.150 2459 DEBUG neutron.agent.linux.utils [req-0fedb962-284b-4bb5-9a08-43deb7994c12 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'dhcp_release', 'tap66fe03bf-ab', '11.100.1.1', 'fa:16:3e:d8:24:a4'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:11:36.166 2459 DEBUG neutron.agent.linux.dhcp [req-0fedb962-284b-4bb5-9a08-43deb7994c12 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Building host file: /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 07:11:36.171 2459 DEBUG neutron.agent.linux.dhcp [req-0fedb962-284b-4bb5-9a08-43deb7994c12 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Done building host file /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 07:11:36.179 2459 DEBUG neutron.agent.linux.utils [req-0fedb962-284b-4bb5-9a08-43deb7994c12 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '10860'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:11:36.190 2459 DEBUG neutron.agent.linux.dhcp [req-0fedb962-284b-4bb5-9a08-43deb7994c12 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Reloading allocations for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 07:11:36.192 2459 DEBUG neutron.agent.linux.utils [req-0fedb962-284b-4bb5-9a08-43deb7994c12 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-4', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:11:36.205 2459 DEBUG neutron.agent.linux.utils [req-0fedb962-284b-4bb5-9a08-43deb7994c12 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5', 'ip', '-6', 'route', 'list', 'dev', 'tap66fe03bf-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:11:36.222 2459 DEBUG neutron.agent.linux.utils [req-0fedb962-284b-4bb5-9a08-43deb7994c12 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:11:36.223 2459 DEBUG neutron.agent.linux.utils [req-0fedb962-284b-4bb5-9a08-43deb7994c12 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:11:36.224 2459 DEBUG neutron.agent.linux.external_process [req-0fedb962-284b-4bb5-9a08-43deb7994c12 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:11:36.224 2459 DEBUG oslo_concurrency.lockutils [req-0fedb962-284b-4bb5-9a08-43deb7994c12 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:12:03.201 2459 DEBUG oslo_concurrency.lockutils [req-b86ff9a1-83fc-4d16-a4c5-50d3ba80540a d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:12:03.202 2459 DEBUG neutron.agent.dhcp.agent [req-b86ff9a1-83fc-4d16-a4c5-50d3ba80540a d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:12:03.205 2459 DEBUG neutron.agent.linux.utils [req-b86ff9a1-83fc-4d16-a4c5-50d3ba80540a d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['kill', '-9', '10860'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:12:03.222 2459 DEBUG neutron.agent.linux.utils [req-b86ff9a1-83fc-4d16-a4c5-50d3ba80540a d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-14 07:12:03.255 2459 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tap66fe03bf-ab) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-14 07:12:03.330 2459 DEBUG neutron.agent.linux.utils [req-71e251bc-4ee2-4610-b92f-87a3c5976366 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:12:03.342 2459 DEBUG neutron.agent.linux.utils [req-71e251bc-4ee2-4610-b92f-87a3c5976366 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:12:03.353 2459 DEBUG oslo_concurrency.lockutils [req-71e251bc-4ee2-4610-b92f-87a3c5976366 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:12:12.365 2459 DEBUG oslo_concurrency.lockutils [req-1be696de-a4b1-42c5-9ae0-f3b204974dd8 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:12:12.386 2459 DEBUG neutron.agent.dhcp.agent [req-14c005aa-07b2-4b57-b14a-b545e459ce0e - - - - -] resync (cd5b296e-c635-4f46-bf81-ce5baa0224c5): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:256
2018-09-14 07:12:12.688 2459 DEBUG neutron.agent.linux.utils [req-a736c5a4-1461-49fd-83c9-80f13716bd67 - - - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:12:12.689 2459 DEBUG neutron.agent.linux.utils [req-a736c5a4-1461-49fd-83c9-80f13716bd67 - - - - -] Unable to access /var/lib/neutron/external/pids/cd5b296e-c635-4f46-bf81-ce5baa0224c5.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:12:12.689 2459 DEBUG neutron.agent.linux.external_process [req-a736c5a4-1461-49fd-83c9-80f13716bd67 - - - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:12:12.690 2459 DEBUG neutron.agent.dhcp.agent [req-a736c5a4-1461-49fd-83c9-80f13716bd67 - - - - -] Calling driver for network: cd5b296e-c635-4f46-bf81-ce5baa0224c5 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 07:12:12.692 2459 DEBUG neutron.agent.linux.utils [req-a736c5a4-1461-49fd-83c9-80f13716bd67 - - - - -] Unable to access /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:12:12.692 2459 DEBUG neutron.agent.linux.utils [req-a736c5a4-1461-49fd-83c9-80f13716bd67 - - - - -] Unable to access /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 07:12:12.693 2459 DEBUG neutron.agent.linux.external_process [req-a736c5a4-1461-49fd-83c9-80f13716bd67 - - - - -] No process started for cd5b296e-c635-4f46-bf81-ce5baa0224c5 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 07:12:12.694 2459 DEBUG neutron.agent.linux.dhcp [req-a736c5a4-1461-49fd-83c9-80f13716bd67 - - - - -] Unable to access /var/lib/neutron/dhcp/cd5b296e-c635-4f46-bf81-ce5baa0224c5/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:271
2018-09-14 07:12:12.694 2459 DEBUG neutron.agent.linux.dhcp [req-a736c5a4-1461-49fd-83c9-80f13716bd67 - - - - -] No interface exists for network cd5b296e-c635-4f46-bf81-ce5baa0224c5 destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1487
2018-09-14 07:12:12.747 2459 DEBUG neutron.agent.linux.utils [req-ff52a133-4fdb-4276-a355-f7c08ac81414 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 07:12:12.763 2459 DEBUG neutron.agent.linux.dhcp [req-ff52a133-4fdb-4276-a355-f7c08ac81414 - - - - -] Namespace already deleted: qdhcp-cd5b296e-c635-4f46-bf81-ce5baa0224c5 _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:249
2018-09-14 07:12:12.764 2459 DEBUG oslo_concurrency.lockutils [req-ff52a133-4fdb-4276-a355-f7c08ac81414 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:12:12.766 2459 INFO neutron.agent.dhcp.agent [req-14c005aa-07b2-4b57-b14a-b545e459ce0e - - - - -] Synchronizing state
2018-09-14 07:12:13.475 2459 INFO neutron.agent.dhcp.agent [req-3c546acc-e1eb-4a50-879b-9afe624663a0 - - - - -] All active networks have been fetched through RPC.
2018-09-14 07:12:13.478 2459 INFO neutron.agent.dhcp.agent [-] Starting network 990b4ddf-dd04-44e0-920c-02fce451b987 dhcp configuration
2018-09-14 07:12:13.480 2459 INFO neutron.agent.dhcp.agent [-] Finished network 990b4ddf-dd04-44e0-920c-02fce451b987 dhcp configuration
2018-09-14 07:12:13.480 2459 INFO neutron.agent.dhcp.agent [-] Starting network cd5b296e-c635-4f46-bf81-ce5baa0224c5 dhcp configuration
2018-09-14 07:12:13.481 2459 INFO neutron.agent.dhcp.agent [-] Finished network cd5b296e-c635-4f46-bf81-ce5baa0224c5 dhcp configuration
2018-09-14 07:12:13.481 2459 INFO neutron.agent.dhcp.agent [req-3c546acc-e1eb-4a50-879b-9afe624663a0 - - - - -] Synchronizing state complete
2018-09-14 07:12:14.009 2459 DEBUG oslo_concurrency.lockutils [req-1be696de-a4b1-42c5-9ae0-f3b204974dd8 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 07:12:14.010 2459 DEBUG oslo_concurrency.lockutils [req-1be696de-a4b1-42c5-9ae0-f3b204974dd8 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Releasing semaphore "dhcp-agent-network-lock-cd5b296e-c635-4f46-bf81-ce5baa0224c5" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 07:12:28.683 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 07:12:28.686 2459 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
